distributed/deploy/tests/test_local.py

```diff
@@ -156,9 +156,9 @@ def test_Client_solo(loop):
 @gen_test()
 def test_duplicate_clients():
     pytest.importorskip('bokeh')
-    c1 = yield Client(processes=False, silence_logs=False, diagnostics_port=9876)
+    c1 = yield Client(processes=False, silence_logs=False, dashboard_address=9876)
```

Comment:
Non semicolon?

Reply:
As above, this should still be valid.  We still support many possible argument types for now.